<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VentanadeConversacionDialog</class>
 <widget class="QDialog" name="VentanadeConversacionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="chatLineEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>9</y>
     <width>461</width>
     <height>21</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white; color: black</string>
   </property>
   <property name="text">
    <string>Chat</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="EnviarpushButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>320</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Enviar!!</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="MensajelineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Ingresa el mensaje que deseas enviar. </string>
   </property>
  </widget>
  <widget class="QTableWidget" name="conversaciontableWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>451</width>
     <height>241</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustIgnored</enum>
   </property>
   <property name="autoScroll">
    <bool>true</bool>
   </property>
   <property name="verticalScrollMode">
    <enum>QAbstractItemView::ScrollPerPixel</enum>
   </property>
   <property name="horizontalScrollMode">
    <enum>QAbstractItemView::ScrollPerPixel</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="BusquedaMensajelineEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Buscar Mensaje</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BuscarpushButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Buscar !</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ConfiguracionesButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>320</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Ajustes</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
